internal/pkg/ociimage/unpack.go

```diff
@@ -74,15 +75,25 @@ func UnpackRootfs(_ context.Context, srcImage v1.Image, destDir string) (err err
 	if os.Geteuid() != 0 || insideUserNs {
 		mapOptions.Rootless = true
 
-		uidMap, err := idtools.ParseMapping(fmt.Sprintf("0:%d:1", os.Geteuid()))
+		uid, err := safecast.ToUint32(os.Getuid())
```

Comment:
Did this intentionally change `Geteuid` to `Getuid`?

Reply:
No - not intentionally. I can change it back at some point, but if uid != euid here then something very odd is going on, and things are going to break later on for sure.